### PR TITLE
Added an exercise/top tips for collaboration

### LINKED ISSUE #233: Condense collaboration recommendations
@jt14den expressed a desire for some of the collaboration material to be condensed down, and I wondered about a "Top Ten Tips"-style list akin to [the list of recommendations for teaching by participatory live demo](https://carpentries.github.io/instructor-training/17-live.html#top-ten-tips-for-participatory-live-coding-in-a-workshop) in the Instructor Training curriculum.

Would anyone else support this idea? Want to get us started on what our top ten tips might be?

### PR DESCRIPTION
**Pull request by anenadic**
Added a 20 minute exercise to reflect on best practices for collaborating on a lesson development project. This means we need to check if this addition fits within the schedule or something needs to be cut down.

Fixes #233.

Noticed some issue with lists within solution div not being renderred as expected. Bullet point lists without a empty like between points is renderred as one big block of text. Numbered lists are not showing numbers. Maybe it is just me missing some knowledge.

### CODE REVIEW

**Comment by anenadic** [comment]
Tagging @jt14den to check/review this too as it was his suggestion to include some kinds of top tips.

File: episodes/wrap-up3.md
Comment (anenadic) [inline episodes/wrap-up3.md:67]:
```suggestion
  up to speed.
```

Comment (anenadic) [reply episodes/wrap-up3.md:67]: Removed repetition

**Comment by sstevens2** [comment]
Will review shortly.  I've had issues with numbered/bulledt lists in that they require a blank line before they start so something like

```
This list contains 3 things...
1. item 1
2. item 2
3. item 3
```

Would not work but...
```
This list contains 3 items...

1. item 1
2. item 2
3. item 3
```
does render.

I've not seen this issue with solution block lists though.  Curious, what happens if you force break the lines with extra spaces at the end.

File: episodes/wrap-up3.md
Comment (sstevens2) [inline episodes/wrap-up3.md:31]:
```suggestion
## Exercise: What steps can you take to be a good collaborator? (20 minutes)
```
Don't think this should be a level 1 header, my understanding it is best practice is that is only used once for the top level of the page.

File: episodes/wrap-up3.md
Comment (sstevens2) [inline episodes/wrap-up3.md:29]: Maybe this should be a discussion block instead of a challenge?  I'm not quite sure myself.

File: episodes/wrap-up3.md
Comment (sstevens2) [inline episodes/wrap-up3.md:42]: I think this block should be in the main text as it is a very important list and I think it would be good not to hide it in the solution box.  Maybe a case for making the challenge a discussion as well and then having the ten tips come right after as their own section with a header.

**Review by sstevens2 — CHANGES_REQUESTED** [review]
This looks like a wonderful addition @anenadic!

I've left a couple of small comments and a suggestion for some rearrangement of the end challenge.
As for time, perhaps the updates you made to the collaboration section with the removal of the project boards will make time already?  Would love to hear what others think.

**Comment by anenadic** [comment]
> Will review shortly. I've had issues with numbered/bulleted lists in that they require a blank line before they start so something like
> 
> ```
> This list contains 3 things...
> 1. item 1
> 2. item 2
> 3. item 3
> ```
> 
> Would not work but...
> 
> ```
> This list contains 3 items...
> 
> 1. item 1
> 2. item 2
> 3. item 3
> ```
> 
> does render.
> 
> I've not seen this issue with solution block lists though. Curious, what happens if you force break the lines with extra spaces at the end?

Will try- thank you!

File: episodes/wrap-up3.md
Comment (anenadic) [reply episodes/wrap-up3.md:31]: Not sure - I copied the exercise block from another exercise and it had the level 1 heading. @tobyhodges  will know better - will check with him.

File: episodes/wrap-up3.md
Comment (anenadic) [reply episodes/wrap-up3.md:42]: That's probably a good idea.

File: episodes/wrap-up3.md
Comment (tobyhodges) [inline episodes/wrap-up3.md:74]:
```suggestion
1. Provide ‘external-facing’ features for your lesson (such as description, objectives, contribution and 'how to get involved'
   guides, preferred way of communication) - these will be universally useful for everyone coming to your repository (your team included).
2. Find the best tools that work for your team for communication, documentation and knowledge sharing, issue & project management.
3. Document and share knowledge within the team to facilitate team collaboration. Share decisions and communicate with your team synchronously and asynchronously.
4. Plan your meetings to get the most out of your team (e.g. set agenda, assign meeting roles - chair, note-taker, time-keeper, etc.) 
   and be appreciative of people's times.
5. Record and manage issues with your lesson as you develop, trial and maintain it (e.g. assign people responsible for 
   issues, categorise and prioritise them).
6. Assign tasks and roles to distribute the responsibilities - this includes assigning larger roles (project management 
   & governance) to team members and assigning specific issues/tasks to collaborators. Progress is much more likely to be made 
   when people know what they and others should be working on.
7. Dedicated sprints where the whole team works on a set of issues towards a milestone work well for lesson development and making progress on projects in general.
8. Be open and welcoming to new contributors and provide different opportunities to get involved with your project 
   (from novice to expert collaborators). Point to upskilling guides (or run upskilling sessions if possible) to bring people 
   up to speed.
9. Manage contributions in a timely manner by noticing when things happen and responding to new issues, comments and 
   Pull Requests (PRs). Show appreciation for everyone's contributions. 
10. People bring different perspectives, expertise and skills to a collaborative project and, while they can be productive 
   and harmoniously complement each other, having some decision making and conflict resolution process in place will make sure 
   progress can be made when disagreements arise.
```

This should fix the formatting issue you've been seeing.

File: episodes/wrap-up3.md
Comment (tobyhodges) [reply episodes/wrap-up3.md:42]: +1

**Review by tobyhodges — COMMENTED** [review]
I agree with @sstevens2's assessment.

In the medium term, I believe it could be better to move this set of tips earlier in the curriculum, as an introduction to the Collaboration Skills episodes, and follow it with some specific demonstrations and deeper dives as needed for each item. (This is what I meant in https://github.com/carpentries/lesson-development-training/issues/233#issuecomment-1717520494 though my explanation was not very clear.) But for now, I think it is good in the wrap-up.

File: episodes/wrap-up3.md
Comment (anenadic) [reply episodes/wrap-up3.md]: I also prefer the list to be in the main body of the episode as it gets lost in the solution but I may have misunderstood some of comments in the original issue #233. And it would also be a nice intro to the collaboration part of the training and then we go and address the tools/guidance to achieve each of the recommendations - I am also OK for it to stay in wrap-up for the time being and we can revisit this (will make sure to open an issue for this).

File: episodes/wrap-up3.md
Comment (anenadic) [reply episodes/wrap-up3.md]: Fixed via 35e6912a.

File: episodes/wrap-up3.md
Comment (anenadic) [reply episodes/wrap-up3.md]: Fixed via 35e6912a.

**Review by anenadic — COMMENTED** [review]
Hopefully have addressed all the requested changes via 35e6912a. Re-requesting review @sstevens2 @tobyhodges .

**Review by tobyhodges — APPROVED** [review]

**Comment by anenadic** [comment]
I cannot merge this PR - it appears some requested changes by @sstevens2 have not been addressed but when I try to see what they are and to address them - they appear as already addressed. Have you seen this @tobyhodges ?

https://github.com/carpentries/lesson-developmentimagetraining/pull/273#pullrequestreview-1626395944

**Comment by tobyhodges** [comment]
That's right: because she requested changes, @sstevens2 must re-review before this can be merged

**Review by sstevens2 — APPROVED** [review]
Looks good to me! Thanks Aleks!

**Pull request merged**
